easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -161,6 +161,13 @@ def configure_step(self):
                  r"\1\n\techo 'libfox: external module used' #"),
             ]
             apply_regex_substitutions('Makefile', regex_subs)
+            # Make configure and make look in the correct directory
+            regex_subs = [
+                (r"\(TOPDIR\)/FoX/finclude",
+                 r"(EBROOTFOX)/finclude"),
+            ]
```

Comment:
Isn't it more clear to collapse this to a single line, since it's only a single substitution?

```python
regex_subs = [(r"\(TOPDIR\)/FoX/finclude", r"(EBROOTFOX)/finclude")]

Reply:
I am just following the same style than above.

Reply:
Well, ok, the one above should be changed too ;)